src/hooks/listHooks.js

```diff
@@ -1,21 +1,20 @@
-/* eslint-disable complexity */
-/* eslint-disable react/prop-types */
 import { isArray } from 'lodash';
 import React, { useCallback, useContext } from 'react';
+import { View } from 'react-native';
 
 import { SettingsContext } from '../SettingsProvider';
-import { SectionHeader, VoucherListItem } from '../components';
+import { ConversationListItem, SectionHeader, VoucherListItem } from '../components';
 import { CardListItem } from '../components/CardListItem';
 import { TextListItem } from '../components/TextListItem';
 import { VolunteerApplicantListItem } from '../components/volunteer/VolunteerApplicantListItem';
 import { VolunteerConversationListItem } from '../components/volunteer/VolunteerConversationListItem';
 import { VolunteerPostListItem } from '../components/volunteer/VolunteerPostListItem';
-import { consts, texts } from '../config';
+import { consts, normalize } from '../config';
 import { momentFormat } from '../helpers';
 import { QUERY_TYPES } from '../queries';
 import { ScreenName } from '../types';
 
-const { LIST_TYPES, ROOT_ROUTE_NAMES } = consts;
+const { LIST_TYPES } = consts;
 
 const getListType = (query, listTypesSettings) => {
```

Comment:
When I click on any date (section header) in the event list, I get an error that the title could not be found which crashes the app. After importing texts I get the error ROOT_ROUTE_NAMES not found. These two errors are just import errors.

|texts error|ROOT_ROUTE_NAMES error|
|--|--|
![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 08 53 32](https://github.com/user-attachments/assets/810c6549-69fb-4049-a090-fd67b8af821b)|![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 08 58 51](https://github.com/user-attachments/assets/21f8205d-9742-439b-a8a6-0b41ad45448c)